Expecto/Expect.fs

```diff
@@ -174,8 +174,8 @@ let isNotWhitespace (actual : string) format =
 let inline equal (actual : 'a) (expected : 'a) format =
   match box actual, box expected with
   | (:? string as a), (:? string as e) ->
-    use ai = a.GetEnumerator()
-    use ei = e.GetEnumerator()
+    let ai = a.ToCharArray().GetEnumerator()
```

Comment:
The `char[].GetEnumerator()` does not implement `IDisposable` and therefore the `use` statement cannot be used here. There is no need for disposal because it's all managed GC memory.